solutions/CF Kayaking.mdx

```diff
@@ -0,0 +1,52 @@
+---
+id: cf-kayaking
+source: CF
+title: Kayaking
+author: Danh Ta Chi Thanh
+---
+
+[Official Editorial](https://codeforces.com/blog/entry/54708)
+
+**Time Complexity:** $\mathcal{O}(N^3)$
+
+<LanguageSection>
+
+<CPPSection>
+
+## Implementation
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+int main() {
+	ios_base::sync_with_stdio(0);
+	cin.tie(0);

```

Comment:
Optional: maybe try putting `cin.tie` together with `ios_base::sync_with_stdio(0);`...

Reply:
I think it is fine so no need to change this.